### PR TITLE
Extract playbook-docs

### PR DESCRIPTION
**Pull request by xjunior**

### CODE REVIEW

**Comment by indiebrain** [comment]
The changes contained in this PR are causing PagerDuty for the on-call team because [the application's eating up 100% of its ephemeral-storage quota](https://metrics.powerapp.cloud/d/6MFyi50Zzw3/kube-quotas-namespace?orgId=1&from=1606863000000&to=1606866600000&var-datasource=prometheus-app-beta-gm&var-typeVirtual=All&var-typePhysical=All&var-nature=All&var-namespace=playbook-pr1225&var-pod=All). It looks like the application in the container is broken and failing to start, incessantly dumping tons of logs to the filesystem:

```
runsv puma: fatal: unable to start ./run: file does not exist
runsv puma: fatal: unable to start ./run: file does not exist
runsv puma: fatal: unable to start ./run: file does not exist
runsv puma: fatal: unable to start ./run: file does not exist
....
```

I'm deprovisioning this PR to stop the alerts.

**Comment by jasoncypret** [comment]
Can you add a description to this PR. My first questions are, what changed, and has the startup changed. These should be added to the description.

File: playbook-docs/README.md [hunk 1]
```diff
@@ -0,0 +1,24 @@
+# README
```
Comment (jasoncypret) [inline playbook-docs/README.md:1]: How has startup changed? Does the old readme need to be moved here?

**Comment by xjunior** [comment]
> Can you add a description to this PR. My first questions are, what changed, and has the startup changed. These should be added to the description.

Hey, @jasoncypret . This PR is WIP. I'll update scripts and README's as the last step cause I'm not sure how far I'm from done yet.

**Comment by xjunior** [comment]
Closing this PR for now as we found deeper coupling from nitro to the docs.  The goal is still the same, but a different strategy will be used.

**Pull request closed without merging**